packages/eslint-config-airbnb-base/flat/rules/errors.js

```diff
@@ -0,0 +1,189 @@
+module.exports = {
```

Comment:
it’s definitely untenable to duplicate the config and make more than one source of truth.

Reply:
It was still an incomplete draft but I give up on the direction I tried.

Reply:
That's fine, but please reuse PRs instead of closing them and opening new ones.